### DISCUSSION TITLE
relevance of hydration `slots` for different frameworks

### DISCUSSION THREAD

**Discussion by codepunkt** [ElMassimo/iles#175] [Q&A] [answered]
[@islands/hydration](https://github.com/ElMassimo/iles/blob/9761eab216d7cbcd2a5bce65003236f99d93b4e4/packages/hydration/hydration.ts) uses something called `slots` to hydrate interactive components. Is this relevant for all frameworks, or only for specific ones (e.g. vue?)

**Comment by ElMassimo** [comment, marked as answer]
All frameworks, although Preact and Solid only have a "default" slot.

Named slots are supported in Svelte and Vue islands, as both frameworks natively support named slots.

That allows an îles user to pass several pieces of content that will be injected in different parts of the island.
